en-US/mozorg/internet-health/includes/newsletter.lang

```diff
@@ -0,0 +1,10 @@
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/includes/newsletter/
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health/includes/newsletter/
+
+
+;Stay informed
+Stay informed
```

Comment:
Drop this file, strings are already covered in the other files in /internet-health